spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/test/java/org/springframework/cloud/dataflow/common/test/docker/compose/execution/DockerTests.java

```diff
@@ -29,60 +29,53 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-public class DockerTests {
+class DockerTests {
 
-    private final DockerExecutable executor = mock(DockerExecutable.class);
-    private final Docker docker = new Docker(executor);
+    private DockerExecutable executor = mock(DockerExecutable.class);
+    private Docker docker = new Docker(executor);
+    private Process executedProcess = mock(Process.class);
 
-    private final Process executedProcess = mock(Process.class);
-
-    @Before
-    public void before() throws IOException {
+    @BeforeEach
+    void prepareForTest() throws IOException {
+        when(executor.commandName()).thenReturn("docker-compose");
         when(executor.execute(any())).thenReturn(executedProcess);
+        when(executor.execute(any(String[].class))).thenReturn(executedProcess);
         when(executedProcess.exitValue()).thenReturn(0);
     }
-
+    
     @Test
-    public void call_docker_rm_with_force_flag_on_rm() throws IOException, InterruptedException {
+    void callDockerRmWithForceFlagOnRm() throws Exception {
         when(executedProcess.getInputStream()).thenReturn(toInputStream(""));
-
         docker.rm("testContainer");
-
         verify(executor).execute("rm", "-f", "testContainer");
     }
 
     @Test
-    public void call_docker_network_ls() throws IOException, InterruptedException {
+    void callDockerNetworkLs() throws Exception {
         String lsOutput = "0.0.0.0:7000->7000/tcp";
         when(executedProcess.getInputStream()).thenReturn(toInputStream(lsOutput));
-
         assertThat(docker.listNetworks(), is(lsOutput));
-
         verify(executor).execute("network", "ls");
     }
 
     @Test
-    public void call_docker_network_prune() throws IOException, InterruptedException {
+    void callDockerNetworkPrune() throws Exception {
         String lsOutput = "0.0.0.0:7000->7000/tcp";
         when(executedProcess.getInputStream()).thenReturn(toInputStream(lsOutput));
```

Comment:
Nitpick: Can we add this mock to the prepareForTest since it is also used in the `callDockerRmWithForceFlagOnRm` test?

Reply:
Sure. I should have added some more description around these method names, but these tests were in very bad shape (I mean snake cases test names in Java LOLWUT?) The remaining of the tests in the module are still like that.

Reply:
Actually, every test does some mocking around this method but the `thenReturn` is different in all except 2 of them. If we pull this up to the the prepare method I think it will be more confusing to follow what each is doing.